tests/test_orient_symmetric.py

```diff
@@ -0,0 +1,82 @@
+from unittest import TestCase
+
+import numpy as np
+from numpy import linalg, random
+
+from aspire.abinitio import CLSymmetryC3C4
+from aspire.source import Simulation
+
+
+class OrientSymmTestCase(TestCase):
+    def setUp(self):
+        L = 32
+        symm = "C4"
+        n_ims = 32
+        src = Simulation(L=L, n=n_ims, symmetry_type=symm)
+        self.cl_class = CLSymmetryC3C4(src, n_symm=4, n_theta=360)
+        self.n_ims = n_ims
+
+    def tearDown(self):
+        pass
+
+    def testGlobalJSync(self):
+        n_ims = self.n_ims
+
+        # Build a set of outer products of random third rows.
+        vijs, viis = self.buildOuterProducts(n_ims)[:2]
+
+        # J-conjugate some of these outer products (every other element).
+        J = np.diag((1, 1, -1))
```

Comment:
isn't this -J?

Reply:
Big oops! hope that doesn't break everything.

Reply:
Turns out `-1 x -1 = 1`, so since we are always J conjugating (ie. J@A@J), it doesn't matter if we use J or -J. For some reason -J is being used in the matlab code.